internal/services/network/public_ip_resource.go

```diff
@@ -107,10 +107,7 @@ func resourcePublicIp() *pluginsdk.Resource {
 				ForceNew: true,
 				Default:  string(publicipaddresses.PublicIPAddressSkuNameStandard),
 				// https://azure.microsoft.com/en-us/updates/upgrade-to-standard-sku-public-ip-addresses-in-azure-by-30-september-2025-basic-sku-will-be-retired/
```

Comment:
Per this link the Basic SKU has been retired. Can you please add appropriate warnings / validation logic to this resource.